File: frontend-angular/src/app/dto/gestionar-conductores/conductor-recibiendo/cConductorRecibiendoDTO.ts
import {CBusEnviandoDTO} from '../conductor-enviando/cBusEnviandoDTO';
import {BRutaEnvioDTO} from '../../gestionar-bus/bus-envio/bRutaEnvioDTO';
import {CDireccionRecibiendoDTO} from './cDireccionRecibiendoDTO';

export class CConductorRecibiendoDTO {
  constructor(
    public id: string,
    public nombre: string,
    public apellido: string,
    public cedula: string,
    public telefono: string,
    public direccion: CDireccionRecibiendoDTO,
    public diasSemana: string[],
    public buses: CBusEnviandoDTO[],
    public rutas: BRutaEnvioDTO[]
  ) {}
}

File: backend-springboot/src/main/java/com/javeriana/pontimovil/ponti_movil/dto/gestionar_conductores/conductor_recibiendo/cConductorRecibiendoDTO.java
package com.javeriana.pontimovil.ponti_movil.dto.gestionar_conductores.conductor_recibiendo;

import com.javeriana.pontimovil.ponti_movil.dto.gestionar_conductores.conductor_enviado.cBusEnviandoDTO;
import lombok.*;

import java.io.Serializable;
import java.util.List;
import java.util.UUID;

@Getter
@Setter
@AllArgsConstructor
@EqualsAndHashCode
@NoArgsConstructor
public class cConductorRecibiendoDTO implements Serializable {
    UUID id;
    String nombre;
    String apellido;
    String cedula;
    String telefono;
    cDireccionRecibiendoDTO direccion;
    List<String> diasSemana;
    List<cBusEnviandoDTO> buses;
}